bears/go/GoErrCheckBear.py

```diff
@@ -40,7 +41,8 @@ def create_arguments(filename, file, config_file,
                              regex apply to all packages.
         :param ignorepkg:    Takes a comma-separated list of package
                              import paths to ignore.
-        :param asserts:      Enables checking for ignored type assertion results
+        :param asserts:      Enables checking for ignored type assertion
+                             results
```

Comment:
could you add a period here in a separate commit please :D ? now i notice it, thanks